### DISCUSSION TITLE
How can I set handlr as the default handler?

### DISCUSSION THREAD

**Discussion by archelaus** [Anomalocaridid/handlr-regex#43] [Q&A]
Currently, when I open a downloaded `.md` file in Firefox, it opens in the default text editor, but I want it to open in nvim. It works as expected if I do `open` or `xdg-open` in my terminal. I've created an `xdg-open` wrapper script in `~/.local/bin` with the contents `handlr open "$@"`

**Comment by archelaus** [comment]
@Anomalocaridid

**Comment by Anomalocaridid** [comment]
Do you have `text/*` or `text/markdown` set to neovim's desktop file? If so, is there a desktop file for neovim in either `/usr/share/applications` or `~/.local/share/applications`?

**Reply by archelaus** [reply]
Yes, I've set `text/*` to nvim. Fwiw I've tried to put my `xdg-open` wrapper in `/usr/local/bin` but no dice.

**Reply by Anomalocaridid** [reply]
Sorry for another slow response. Is it set to the `nvim` command or to neovim's desktop file? Does the `text/*` entry show up when you run `handlr list`?
